forest/main.py

```diff
@@ -205,14 +205,14 @@ def on_change(attr, old, new):
         step=0.1,
         value=1.0,
         show_value=False)
-    custom_js = bokeh.models.CustomJS(
-            args=dict(renderers=renderers),
-            code="""
-            renderers.forEach(function (r) {
-                r.glyph.global_alpha = cb_obj.value
-            })
-            """)
-    slider.js_on_change("value", custom_js)
+    # custom_js = bokeh.models.CustomJS(
+    #         args=dict(renderers=renderers),
+    #         code="""
+    #         renderers.forEach(function (r) {
+    #             r.glyph.global_alpha = cb_obj.value
+    #         })
+    #         """)
+    # slider.js_on_change("value", custom_js)
```

Comment:
This should be reactivated by selecting "image" renderers, since RDT doesn't have an alpha value